index.bs

```diff
@@ -5,9 +5,9 @@ Status: ED
 Shortname: longtasks
 Level: 1
 URL: https://w3c.github.io/longtasks/
-Editor: Shubhie Panicker, Google https://google.com, panicker@chromium.org
-Editor: Ilya Grigorik, Google https://google.com, igrigorik@chromium.org
-Editor: Domenic Denicola, Google https://google.com, domenic@chromium.org
+Editor: Shubhie Panicker, Google https://google.com, panicker@chromium.org, w3cid 92587
```

Comment:
I think we can move these folks to former editors and add @noamr as an editor, as he's working on extending LT with LoAF